### DISCUSSION TITLE
https://somiibo.com/blog/best-twitter-bot/

### DISCUSSION THREAD

**Discussion by giscus** [Distalk/giscus#7] [Giscus]
# https://somiibo.com/blog/best-twitter-bot/

Somiibo lets you enter specific hashtags you want to be associated with and then it will automatically like tweets and follow users based on those custom hashtags...

https://somiibo.com/blog/best-twitter-bot/

**Comment by livenecktie** [comment]
Their support is good. You should recommend it [slope unblocked](https://slope-unblocked.co).
